File: ryan-howard/ConvertStringToCamelCase.js
function toCamelCase(str){
  str = str.split('');
  return str.map(function(el,i) {
    if(el == '-' || el == "_") {
      el = str[i+1].toUpperCase();
      str.splice(i+1, 1);
    }
    return el;
  }).join('');
}

//take string, split it into an array of characters 
// make an if else to change the text or get rid of dashes/underscores
// put a map in the if else to change if it has a - or _ and then later join
// make it so the next letter following a dash or underscore is converted toUpperCase

// had to look up the splice (i+1, 1) the comma one was missing

File: ryan-howard/ConvertStringToCamelCase.py
def to_camel_case(text):
    words = text.replace('-', '_').split('_')
    return words[0] + ''.join(word.capitalize() for word in words[1:])


#get rid of dashes and underscores
#capitalize first Letter of stealth and warrior
#split each word, capitalize the first letter of each one, and join them together
